### PR TITLE
Add jupyter notebook/lab plotting deps in README

### PR DESCRIPTION
**Pull request by 2timesjay**
Address open issue about missing plotting details.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -130,6 +130,14 @@ If using Ax in Jupyter notebooks:
 pip3 install git+ssh://git@github.com/facebook/Ax.git#egg=Ax[notebook]
 ```
 
+To support plotly-based plotting in newer Jupyer notebook versions
```
Comment (kkashin) [inline README.md]: Jupyer -> Jupyter

File: README.md [hunk 2]
```diff
@@ -130,6 +130,14 @@ If using Ax in Jupyter notebooks:
 pip3 install git+ssh://git@github.com/facebook/Ax.git#egg=Ax[notebook]
 ```
 
+To support plotly-based plotting in newer Jupyer notebook versions 
+
+```
+pip install "notebook>=5.3" "ipywidgets==7.5"
```
Comment (kkashin) [inline README.md]: Should we also mention that Plotly >= 4.0?

File: README.md [hunk 3]
```diff
@@ -130,6 +130,14 @@ If using Ax in Jupyter notebooks:
...
+```
+
+[See plotly repo's README](https://github.com/plotly/plotly.py#jupyter-notebook-support) for details and Jupyterlab instructions.
```
Comment (kkashin) [inline README.md]: Nit: 

```suggestion
[See Plotly repo's README](https://github.com/plotly/plotly.py#jupyter-notebook-support) for details and JupyterLab instructions.
```

File: README.md (hunk 2 above)
Comment (2timesjay) [reply README.md]: Is it strictly necessary? I didn't think we were compatible

**Pull request closed without merging**